core/trie.go

```diff
@@ -0,0 +1,73 @@
+package core
+
+import (
+	"strconv"
+
+	"github.com/multiversx/mx-chain-core-go/core/check"
+)
+
+// TrieNodeVersion defines the version of the trie node
+type TrieNodeVersion uint8
+
+const (
+	// NotSpecified means that the value is not populated or is not important
+	NotSpecified TrieNodeVersion = iota
+
+	// AutoBalanceEnabled is used for data tries, and only after the activation of AutoBalanceDataTriesEnableEpoch flag
+	AutoBalanceEnabled
+)
+
+// GetStringForVersion returns the string representation of the given trie node version
+func GetStringForVersion(version TrieNodeVersion) string {
```

Comment:
For easier integration, can we move the code of this function on the TrieNodeVersion and call the new function `String() string` ?